tests/unit/preprocessor/_area/test_area.py

```diff
@@ -377,6 +378,40 @@ def square_composite_shape(request, tmp_path):
     return np.ma.masked_array(vals, mask)
 
 
+def _create_sample_full_cube():
+    cube = Cube(np.zeros((24, 180, 360)), var_name='co2', units='J')
```

Comment:
```suggestion
    cube = Cube(np.zeros((2, 180, 360)), var_name='co2', units='J')
```
It would be nicer to use a smaller cube to avoid slowing down the tests and making them require a lot of memory

Reply:
good point!